### DISCUSSION TITLE
Increase the task queue for bots.

### DISCUSSION THREAD

**Discussion by local-interloper** [FakeFishGames/Barotrauma#12896] [Ideas and suggestions]
# My Issue:
Telling your friend to do something is way easier than telling a bot to do so. The current command queue system is great but I think it's not robust enough especially for singleplayer where you have no other players to depend on. 

# My potential solution:
It would be nice if we could queue up more than just 3 commands at the time. Or have a Rimworld-esque task priority system. I'd say just expanding the max queue size would do wonders for now.

**Comment by BlackMoons** [comment]
3 works 'okish' until you want to give them an order to follow you or something and now you've got to lose an order and reassign it after deleting the follow me/return to sub/etc command.

So at the very least, 3 orders + a movement, and ideally being able to order the movement before (to pause all orders) / after orders (to idle somewhere)
